### ISSUE TITLE
v0.3.0 release failed on registry

### ISSUE THREAD

**Issue by kdpsingh** [TidierOrg/TidierPlots.jl#29]
I'm not 100% sure from the reading the logs why this failed.

From my quick scan of https://github.com/JuliaRegistries/General/actions/runs/6290588038/job/17078147947?pr=92098, I see:

```
  1 dependency had warnings during precompilation:
1738
┌ MKL_jll [856f044c-d86e-5d09-b602-aeab76dc8ba7]
```

Feel free to keep working on the package. We shouldn't bump the version any further until this is fixed. Once we fix, we will need to update the 0.3.0 release.

And I'll plan to fix the TagBot so that it will automatically take care of the releases.

**Comment by rdboyes** [comment]
It looks like I accidentally added some unneeded dependencies when I was working on support for color scales. I removed them in the latest commit - should work now, I think?

**Comment by kdpsingh** [comment]
Bumping the version between commits can prevent auto-merging. I would recommend changing the version back to 0.3.0 and then tagging the registrator bot. Happy to fix this today if you'd like me to try.

**Comment by kdpsingh** [comment]
We can then delete the newer releases on GitHub. The releases aren't mandatory anyway but good practice.

**Comment by rdboyes** [comment]
I think I have it fixed now? Version is back to 0.3.0 and the PR merged

**Comment by kdpsingh** [comment]
Yup, it looks like everything worked. Once we fix the TagBot, it'll handle the GitHub releases so you won't need to do any of that manually.
